src/hash_set/managed.zig

```diff
@@ -23,23 +23,37 @@ const std = @import("std");
 const mem = std.mem;
 const Allocator = mem.Allocator;
 const SetUnmanaged = @import("unmanaged.zig").HashSetUnmanaged;
+const SetUnmanagedWithContext = @import("unmanaged.zig").HashSetUnmanagedWithContext;
 
 /// fn HashSetManaged(E) creates a set based on element type E.
 /// This implementation is backed by the std.AutoHashMap implementation
 /// where a Value is not needed and considered to be void and
 /// a Key is considered to be a Set element of type E.
 /// The Set comes complete with the common set operations expected
 /// in a comprehensive set-based data-structure.
+/// Calls to HashSetManaged default to a max_load_percentage of 100%.
 pub fn HashSetManaged(comptime E: type) type {
+    return HashSetManagedWithContext(E, void, 100);
```

Comment:
Curious why we choose to default on 100%? User's might see a performance issue when adding items which would cause a lot of regrowth and copying since there is no padding for load. 

Aren't most load factors around 75 to 85 percent? I'm just speaking anecdotally and going off of historical knowledge. Maybe I have a it backwards in my mind and a load factor of 100% would keep the hashset with a whole 100% room to grow before falling back to allocating and copying everything over internally.

Reply:
Yes correct, something like that. I've seen 65-75% but depends on the use case for sure.

I think what I failed to communicate is that if `Context` is void the load percentage can be anything as it's not actually utilised. The initialiser will use an auto hashmap implementation (the default one). In it the load factor cannot be configured so the built-in defaults will be used. I think it's 75%.

I improved this so that the load factor is given as `undefined` if no custom context is set. Hopefully clarifies this! https://github.com/deckarep/ziglang-set/pull/13/commits/1cb1d6f1003efa9acb8edac0e3a2d9a17c836eba